src/Agent.Listener/LocalRunner.cs

```diff
@@ -113,14 +158,8 @@ public sealed override void Initialize(IHostContext hostContext)
 
         private async Task<List<JobInfo>> ConvertToJobMessagesAsync(Pipelines.Process process, CancellationToken token)
         {
-            // Verify the current directory is the root of a git repo.
-            string repoDirectory = Directory.GetCurrentDirectory();
-            if (!Directory.Exists(Path.Combine(repoDirectory, ".git")))
-            {
-                throw new Exception("Unable to run the build locally. The command must be executed from the root directory of a local git repository.");
-            }
-
             // Collect info about the repo.
+            string repoDirectory = Directory.GetCurrentDirectory();
```

Comment:
i'm going to change this to pass this variable in (a little bit cleaner since i moved the .git folder validation out of this method)